qutip/core/cy/qobjevo.pyx

```diff
@@ -292,9 +292,31 @@ cdef class QobjEvo:
 
         return out
 
-    def __call__(self, double t, dict args=None):
-        if args:
-            return QobjEvo(self, args=args)(t)
+    def __call__(self, double t, dict _args=None, **kwargs):
+        """
+        Get the :cls:`Qobj` at ``t``.
+
+        parameter
+        ---------
+        t : float
+            Time at which the ``QobjEvo`` is to be evalued.
+
+        args : dict [positional / optional]
+            New arguments as a dict. Update args with ``arguments(new_args)``.
```

Comment:
```suggestion
        _args : dict [positional / optional]
            New arguments as a dict. Update args with ``arguments(_args)``.
```
Given that the argument is actually `_args` I would include it in that way in the docstring.

Reply:
```suggestion
        _args : dict [optional]
            New arguments as a dict. Update args with ``arguments(new_args)``.
```